lib/pronto/formatter/pull_request_formatter.rb

```diff
@@ -0,0 +1,87 @@
+module Pronto
+  module Formatter
+    class PullRequestFormatter
```

Comment:
Since this code can almost entirely be re-used for the commit formatters maybe this would be better as: `GitClientFormatterBase` or some variation of that.  Thoughts?